### ISSUE TITLE
The generated password string has no any symbols

### ISSUE THREAD

**Issue by saidbakr** [imcoddy/passwork#1]
The generated password string is, always, generated with no any special characters.

**Comment by imcoddy** [comment]
Hi, this is because Bitcoin address doesn't contain any special characters. 
And this app was developed 5 year ago, it's better to use a password manager such as 1Password or Bitwarden for better security.

**Comment by saidbakr** [comment]
I don't like passwords managers. The only password manager that I trust is my securely hidden paper and pencil. The secret that is spread among more than two, it does no longer a secret!

**Comment by imcoddy** [comment]
Yeah that's also why I created this tool at that time. But honestly, recently I feel it hard to manage/update across multiple websites.
I do recommend you host a Bitwarden docker of your own, as it's open source and free
